### PR TITLE
📖 Creating Templates for new feature and bug reporting

### PR DESCRIPTION
**Pull request by joshbermanssw**
**Context:**
Templates improve readability of issues, and allow for the backlog to be concise and well traversable. Having uniform ways to publish issues (of different types) ensures that all relevant information is included for issues. 

**Screenshots:**

Screenshot 2024-04-26 at 11 42 56 am
**Figure: New Feature Issue Template**

Screenshot 2024-04-26 at 11 43 03 am
**Figure: New Bug Issue Template**

### All New Content Submissions: (To be confirmed by reviewer)

* [x] Title is short & specific
* [ ] Headers are logically ordered & consistent
* [ ] Purpose of document is explained in the first paragraph
* [ ] Procedures are tested and work
* [ ] Any technical concepts are explained or linked to
* [ ] Document follows structure from templates
* [ ] All links work
* [ ] The spelling and grammar checker has been run
* [ ] Graphics and images are clear and useful
* [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-bug.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<!-- Add the "contract" that defines the requirements for the GitHub issue to be completed as per the team's agreement -->
+
+### ➕ More Information
+<!-- Add any other context here -->
+
+### 📸 Screenshots 
+<!-- If applicable, add screenshots that are relevent to the feature (i.e mock-ups, diagrams) -->
+
+<!-- Below the Cc, @ mention users who should be in the loop -->
+Cc:
```
Comment (NadeekaJK) [inline .github/ISSUE_TEMPLATE/new-bug.md:37]: Please update the screenshots in the PR description to match the template.

File: .github/ISSUE_TEMPLATE/new-feature.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+### 💼 Tasks 
+<!-- Add GitHub tasks in a measurable, check-box manner -->
+- [ ] ... 
+- [ ] ... 
+
+### ✅ Acceptance Criteria 
+<!-- Add the "contract" that defines the requirements for the GitHub issue to be completed as per the team's agreement -->
+
+### ➕ More Information
+<!-- Add any other context here -->
```
Comment (NadeekaJK) [inline .github/ISSUE_TEMPLATE/new-feature.md:25]: Expand a bit more on the info here.

File: .github/ISSUE_TEMPLATE/new-bug.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+about: Create a report for any found bugs.
+labels: 'Type: Bug'
+title: "🐛 "
+---
+
+<!-- **Tip:** Delete parts that are not relevent -->
+
+### 🐛 Desrcibe the Bug
+<!-- A clear and concise description of what the bug is -->
+
```
Comment (NadeekaJK) [inline .github/ISSUE_TEMPLATE/new-bug.md:12]: Add a priority level.

**Review by NadeekaJK — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/new-bug.md (hunk 3 above)
Comment (joshbermanssw) [reply .github/ISSUE_TEMPLATE/new-bug.md:12]: I believe priority level should be a label rather than in the content of the issue

**Review by NadeekaJK — APPROVED** [review]

**Pull request merged**